AGSnapshotHelper.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
     s.platform         = :ios, '8.0'
     s.requires_arc     = true
 
-    s.source_files     = 'Pod/Classes/**/*.{h,m}'
+    s.source_files     = 'Pod/Classes/**/*.{h,m,swift}'
```

Comment:
In Cocoapods `1.4.0` we can specify supported Swift version in `Podfile` instead of separate file. Maybe it's time to adjust to that?

Reply:
You mean `swift_version` parameter, yes?

Reply:
[Yes.](http://blog.cocoapods.org/CocoaPods-1.4.0/)

Reply:
Done.